.pre-commit-config.yaml

```diff
@@ -18,7 +18,7 @@ repos:
           files: \.(cu|cuh|h|cc|inl)$
           types_or: []
     - repo: https://github.com/pre-commit/mirrors-mypy
-      rev: 'v0.982'
+      rev: 'v0.990'
```

Comment:
FWIW I think 0.991 is also available now

Reply:
https://github.com/nv-legate/legate.core/pull/482/commits/42e4de630424285b43fcd3757c6bdfb35cee53ed